config/dependency-injection/loader-pass.php

```diff
@@ -54,7 +54,7 @@ private function process_definition( Definition $definition, Definition $loader_
 
 		try {
 			$reflect = new ReflectionClass( $class );
-			$path    = $reflect->getFileName();
+			$path    = $this->normalize_slashes( $reflect->getFileName() );
 			if ( strpos( $path, 'wordpress-seo/src/helpers' )
 				|| strpos( $path, 'wordpress-seo/src/actions' )
 				|| strpos( $path, 'wordpress-seo/src/builders' )
```

Comment:
I'd like to ask @thijsoo if there is a chance to improve the hardcoded plugin directory name too...